docs/source/how-to/index.rst

```diff
@@ -73,8 +73,30 @@ The Olive CLI provides a set of primitives such as `quantize`, `finetune`, `onnx
 
       :octicon:`arrow-right;1em;sd-text-info` `olive run <cli/cli-run.html>`_
 
-Configure Workflows (Advanced)
-------------------------------
+How to prepare configuration
+----------------------------
+
+.. toctree::
+   :maxdepth: 2
+   :hidden:
+
+   configure-workflows/build-workflow
+
+For more complex scenarios, you can create a new configuration from scratch.
+
+.. grid:: 2 2 2 3
+   :class-container: cards
+
+   .. grid-item-card::
+
```

Comment:
Would it help with readability and authoring ease by providing examples in yaml?